### ISSUE TITLE
Gulp task loader fails if paths is given with different then OS path separator.

### ISSUE THREAD

**Issue by arnas** [hontas/gulp-task-loader#29]
For example if you provide path 'gulp-task/tasks' it will fail on windows as it uses different paths. Maybe there should be recommendation to pass paths using path library? . As you are using it internally in your code.

**Comment by hontas** [comment]
@arnas could you please provide an example?

**Comment by arnas** [comment]
yeah, sure. It may not be exact case that I encountered,. So for example if I use your module like this.
require('gulp-task-loader')('path/to'); It won't work on windows, as windows requires backward slash. 
Maybe you should leave a note about this. That it's recommended that path would be provided using path.join.

**Comment by hontas** [comment]
Fair enough... but better to just make it work. I'll look in to it.
